app/views/notices/_works_urls.html.erb

```diff
@@ -2,8 +2,11 @@
   <div class="row">
     <span class="label original-title"><%= original_title %></span>
     <ol class="list original-urls">
-      <%# Don't do code style changes here, it's intentional to avoid additional spaces in the response %>
-      <% work.copyrighted_urls.each do |url| %><%= content_tag_for(:li, url) do %><%= url.url %><% end %><% end %>
+      <% if work.copyrighted_urls.each do |url| %>
```

Comment:
We have notices with 10k+ urls, it will cause sending back tons of useless whitespace.

Reply:
I like this change, but it makes sense to, at the same time, use something like this https://thoughtbot.com/blog/content-compression-with-rack-deflater to compress the output.

Reply:
Yeah, and we should be doing that anyway. I'm going to leave this as-is and add a ticket for future work of that nature.